src/routes/game/components/SevenDoubleJacksDialog.vue

```diff
@@ -45,8 +45,8 @@
 </template>
 
 <script>
-import BaseDialog from '@/components/Global/BaseDialog.vue';
-import GameCard from '@/components/GameView/GameCard.vue';
+import BaseDialog from '~core/components/BaseDialog.vue';
+import GameCard from './GameCard.vue';
```

Comment:
Let's stick to imports from `'@'` for flexibility in case files are moved around